iXBRLViewerPlugin/iXBRLViewer.py

```diff
@@ -380,6 +403,7 @@ def createViewer(self, scriptUrl=DEFAULT_VIEWER_PATH, useStubViewer=False, showV
         self.taxonomyData["prefixes"] = self.nsmap.prefixmap
         self.taxonomyData["roles"] = self.roleMap.prefixmap
         self.taxonomyData["rels"] = self.getRelationships()
+        self.taxonomyData["units"] = self.unit_map
```

Comment:
Why do we need more data in the JSON?  Can we build this map in the JavaScript instead?

Reply:
The JSON currently only has a simplified measure attribute (by taking the first numerator from the unit). This PR retains that measure attribute but adds the complex form of the unit.

Reply:
Do we still need the first numerator somewhere in ixbrl-viewer?  It seems like we should add full support for units, and then it won't be required any more.

If it is still required, I think the JavaScript should do that, and the serialized format should be complete.  I don't think it's useful to duplicate the unit data in two JSON keys.

Reply:
Initially I didn't have a grasp on where all the "measure" was used and wasn't confident that I could remove it cleanly, so opted to add unit alongside it. Reconsidering now though, it doesn't look like it will be very problematic to remove measure entirely so I'll take a shot at it.